stock_measuring_device_zippcube/models/zippcube_device.py

```diff
@@ -0,0 +1,109 @@
+# Copyright 2021 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html)
+import logging
+
+from odoo import _, fields, models
+from odoo.exceptions import UserError
+
+_logger = logging.getLogger(__name__)
+
+
+class ZippcubeDevice(models.Model):
+    _name = "zippcube.device"
+    _description = "Measuring Device Zippcube"
+    _order = "warehouse_id, name"
+
+    name = fields.Char("Name", required=True)
```

Comment:
almost everything here can stay in a mixin and/or common model.
You can have a `device_type_id` and based on the details of the type you can provide all the configuration you need (name, description, wizard reference, etc)